libs/libcommon/src/libcommon/queue.py

```diff
@@ -273,17 +276,32 @@ def update_metrics_for_type(job_type: str, previous_status: str, new_status: str
         increase_metric(job_type=job_type, status=new_status)
 
 
+class _TTL(IntEnum):
+    LOCK_TTL_SECONDS_TO_START_JOB = LOCK_TTL_SECONDS_TO_START_JOB
+    LOCK_TTL_SECONDS_TO_WRITE_ON_GIT_BRANCH = LOCK_TTL_SECONDS_TO_WRITE_ON_GIT_BRANCH
+
+
 class Lock(Document):
     meta = {
         "collection": QUEUE_COLLECTION_LOCKS,
         "db_alias": QUEUE_MONGOENGINE_ALIAS,
         "indexes": [
             ("key", "owner"),
             {
+                "name": "LOCK_TTL_SECONDS_NO_OWNER",
```

Comment:
Maybe, it is needed to remove the old TTL (updated_at_1) right? not sure if a migration will work or a manual deletion from db.